packaging_automation/templates/citus.spec.tmpl

```diff
@@ -2,6 +2,7 @@
 %global pgpackageversion 11
 %global pginstdir /usr/pgsql-%{pgpackageversion}
 %global sname citus
+%global debug_package %{nil}
```

Comment:
do we really need this ?

Reply:
ping @gurkanindibay

Reply:
Yes without it we build fails. It is from enterprise config file 
You can check it here
https://github.com/citusdata/packaging/blob/0370f3337e160587d698358ab3c28f0271fde13c/citus-enterprise.spec#L6

Reply:
and this one
https://github.com/citusdata/packaging/blob/bc964730234f14bb5e72600b219b2403f6adc87b/citus.spec#L5

Reply:
Sorry I missed this one